demisto_sdk/commands/common/tools.py

```diff
@@ -232,12 +233,23 @@ def get_remote_file(
                 res = requests.get(github_path, verify=False, timeout=10)
                 # And maybe it's just not defined. 😢
                 if not res.ok:
-                    raise DemistoException(
-                        f'You are working in a private repository: "{githhub_config.CURRENT_REPOSITORY}".\n'
-                        f'Please define your github token in your environment.\n'
-                        f'`export {githhub_config.Credentials.ENV_TOKEN_NAME}=<TOKEN>`',
-                        LOG_COLORS.RED
-                    )
+                    if not suppress_print:
+                        print_warning(
+                            f'You are working in a private repository: "{githhub_config.CURRENT_REPOSITORY}".\n'
+                            f'The github token in your environment is undefined.\n'
+                            f'Getting file from local repository instead. \n'
+                            f'If you wish to get the file from the remote repository, \n'
+                            f'Please define your github token in your environment.\n'
+                            f'`export {githhub_config.Credentials.ENV_TOKEN_NAME}=<TOKEN>`'
+                        )
+                    # Get from local git origin/master instead
+                    repo_name = github_config.CURRENT_REPOSITORY
+                    path_to_dir = os.path.join(repo_name, os.path.dirname(full_file_path))
+                    repo = git.Repo(path_to_dir, search_parent_directories=True)
+                    repo_path = repo.git.rev_parse('--show-toplevel')
```

Comment:
might be right to put these as functions in `git_util` and then use it from there.